src/drivers/mct/cime_config/buildnml

```diff
@@ -23,6 +23,24 @@ from CIME.XML.files import Files
 logger = logging.getLogger(__name__)
 
 ###############################################################################
+def _get_time_in_seconds(time, unit):
```

Comment:
Isn't there already a function in CIME.utils for this?  If not wouldn't that be a better place for it?  Perhaps all of the time utilities should be moved to a new time_utils.py file.